.github/workflows/build.yml

```diff
@@ -37,8 +35,17 @@ jobs:
         with:
           node-version: '14.15.0'
 
-      - name: Build
-        run: node ./run dist --skip-version-validation
+      - name: Build (Ubuntu)
+        run: node ./run dist --skip-version-validation --target linux
+        if: matrix.os == 'ubuntu-latest'
+
+      - name: Build (Windows)
+        run: node ./run dist --skip-version-validation --target win
+        if: matrix.os == 'windows-latest'
+
+      - name: Build (MacOs)
+        run: node ./run dist --skip-version-validation --target macos
+        if: matrix.os == 'macos-latest'
```

Comment:
out of curiosity, why we want to run this job 3 times instead of just 1, on MacOS with cross compilation? IT takes much more CI resources this way.

Reply:
I'm not sure, the CI job was set up this way and I followed the existing setup.

It might better isolate the jobs in case weird things happen due to caching?